synapse_data_warehouse/synapse_raw/tables/V2.23.0__add_version_history_to_nodesnapshot.sql

```diff
@@ -0,0 +1,9 @@
+-- Configure environment
+USE ROLE sysadmin;
```

Comment:
These role and warehouse specifications are redundant since they are specified in the [CI job](https://github.com/Sage-Bionetworks/snowflake/blob/6613b12340f96dd16832da387690bdcccf8c5049/.github/workflows/ci.yaml#L60).

I'm also not a fan of making these specifications in the scripts themselves since this opens up potential inconsistencies from script to script. I think it would make more sense for the job running `schemachange` to be the final source of truth for these configurations. What do you think?

Reply:
I agree. The role should be handled by the deployer, rather than the script itself. If we need to specify _who_ has OWNERSHIP of _what_ that should be handled explicitly in the script.